spec/system/admin_customize_themes_spec.rb

```diff
@@ -83,4 +95,18 @@
       expect(ace_content.text).to eq("console.log('test')")
     end
   end
+
+  describe "when editing theme translations" do
+    it "should allow admin to edit the theme translations" do
+      visit("/admin/customize/themes/#{theme.id}")
+
+      theme_translations_settings_editor =
+        PageObjects::Components::AdminThemeTranslationsSettingsEditor.new
+
+      theme_translations_settings_editor.fill_in("Hello World")
+      theme_translations_settings_editor.save
+
+      expect(theme.reload.translations["Hello World"]).to eq("Hello World")
```

Comment:
One tip for running system tests is this: 

- use `SELENIUM_HEADLESS=0 bin/rspec spec/system/admin_customize_themes_spec.rb:104` to run just the test in that line and have Chrome be headful, i.e. so you can inspect it
- add `pause_test` before an assertion to see the state of the test in that particular moment